components/home/Home.jsx

```diff
@@ -1,10 +1,85 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
-
-export default (props) => {
-   return (    
-      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-         <Text>Home Screen</Text>
-         <Text>Replace these Text tags with your component</Text>
-      </View>);
-}
\ No newline at end of file
+import { Text, View, Button, StyleSheet } from 'react-native';
+import { ScreenStackHeaderCenterView } from 'react-native-screens';
+
+export default class App extends React.Component {
+
+   render() {
+      return (
+         <View style = {styles.container}>
+
+
+            {/*Cycles Section*/}
+            <View style = {[{width: "95%", height: "40%", margin: 30, backgroundColor: "F6F6F6", borderWidth: 5, borderRadius: 15, borderColor: "#5DB075"}]}>
+               <Text style = {styles.title}>
+                  Cycles
+               </Text>
+               <View style = {[{width:"85%", height: "45%", backgroundColor: "#5DB075", borderRadius: 15, borderWidth: 3, borderColor: "#5DB075", position: "absolute", top: 75, alignSelf: "center"}]}>
+                  <Button
+                     title = "Start Workout for Today's Cycle"
+                     color = "#FFFFFF"
+                     onPress = {() => alert("This should start the current workout in the cycle")}
+                  />
+               </View>
+               <View style = {[{width:"85%", borderRadius: 15, borderWidth: 3, borderColor: "#5DB075", position: "absolute", bottom: 20, alignSelf: "center"}]}>
+                  <Button
+                     title = "See all cycles"
+                     color = "#000000"
+                     onPress = {() => alert("This should take you to workouts tab")}
+                  />
+               </View>
+            </View>
+
+
+            {/*Workouts Section*/}
+            <View style = {[{width: "95%", height: "40%", margin: 10, backgroundColor: "#F6F6F6", borderWidth: 5, borderRadius: 15, borderColor: "#5DB075"}]}>
+               <Text style = {styles.title}>
+                  Workouts
+               </Text>
+               <Text style = {[{position: "absolute", top: 82, left: 10, fontSize: 18}]}>
+                  Workout 1 
```

Comment:
This will be a variable name that we get from our database right? you could represent this with like test data for now (declare a variable at the top of your code and fill it temporarily). Then you can just render the variable here instead of "workout 1"

Reply:
Also, are we always going to display just two workouts here? What if the user only has created one workout? We might want to make this part dependent on a list of workout objects

Reply:
Yes, it will later be replaced with a database query. As per your suggestion, I declared 2 variables at the top for this temporarily

Reply:
Yeah my idea was that for now, we just always display 2. However, when we do have a list of workouts per user, it should be different based on how many workouts exist. I think in the workouts tab, if they have more than 2 workouts, there should be an option to change which ones are on the home page. If there is only 2 workouts both show on the home page. If there is only 1 workout, it will show the first one and the second one is replaced with "Make another workout in the workouts tab to show up here" or something.